### ISSUE TITLE
Some stems should not be stems

### ISSUE THREAD

**Issue by eddieantonio** [UAlbertaALTLab/plains-cree-fsts#9]
Like for example, "kinanâskomitin" will be inflected further @atticusha.

@aarppe: can you change the script that generates the stems? How?

https://github.com/UAlbertaALTLab/plains-cree-fsts/blob/07fade275b470355f649f6fd89b74527f1593bee/src/morphology/stems/verb_stems.lexc#L2318

**Comment by atticusha** [comment]
UGH. this is included as a thanking particle, essentially. We could probably redirect it to the IPC lexicon. We should have nanâskomêw in the stems anyways.

**Comment by eddieantonio** [comment]
Hey @aarppe, is this something that should be fixed in the scripts that convert Arok's content to the stems lexc files?

**Comment by aarppe** [comment]
Yes, though what information to base the determination of whether a lexical entry is a specific inflected form or a lemma needs some exploration. Solely AEW's pos-code is not sufficient, but the grammatical information might provide sufficient clues, cf.

`cat ~/Google\ Drive/CreeFST/Wolvengrey/Wolvengrey_eng2crk.csv | gawk -F"\t" '$0 ~ /1s-2s/'
`
**kinanâskomitin**	ᑭᓇᓈᐢᑯᒥᑎᐣ	VTA-1	thank you, I am grateful to you	Independent, 1s-2s

`cat ~/Google\ Drive/CreeFST/Wolvengrey/Wolvengrey_eng2crk.csv | gawk -F"\t" '$0 ~ /2s-1s/'`

**kitatamihin**	ᑭᑕᑕᒥᐦᐃᐣ	VTA-1	thank you, you make me smile, you please me	Independent; 2s-1s

`cat ~/Google\ Drive/CreeFST/Wolvengrey/Wolvengrey_eng2crk.csv | gawk -F"\t" '$0 ~ /2p/'`

**kôhkomipaninaw**	ᑰᐦᑯᒥᐸᓂᓇᐤ	NA-2	"cucumber; literally: ""our deceased grandmother"""	singular; 2p possessive, absentative

**tânitahtwasiyêk**	ᑖᓂᑕᐦᑕᐧᓯᔦᐠ	IPC	there are how many of you (in the family)?	Conjunct, 2p

**Comment by aarppe** [comment]
Note to self - the more major fixing that is needed for the LEXC code generation script is modifying it to work with Arok's new inflectional coding (primarily for verbs). The primary "pain" in this is checking that this new coding is systematic, and incorporating known (and new) exceptions, which have been hand-coded in the current script.

**Comment by katieschmirler** [comment]
Also having some issues with diminutive VTAs: while these technically have a VTA as their stem (before diminutivization), including this in the stems files means that I just saw 80 instances of the following kind of issue in the Bloomfield texts:

_pakahamahwêw_     pakahamahwêsiw+V+TA+Ind+Prs+3Sg+4SgPlO

Diminutive VTAs are their own mess altogether, but this kind of analysis is entirely unhelpful!

**Comment by aarppe** [comment]
Do you mean that the VTA stem in the FST doesn't have incorporate the diminutive, and is only the pre-diminutivized VTA stem.

For the record, for lexicalized cases, the FST always requires the full stems with all derivational processes having been applied to. The FST can in principle recognized productive derivations, but then the stem/lemma that is given is the pre-derivation one.

For the cases above, the LEXC representation of the stems would be the following:

`pîhcikwahêw:pîhcikwah VTA ;`

But what is the third person suffix in the following forms: _pa-pakamahwêsiw_ _pakamahwêsiw_? It's not _-iw_, and there's not _-êw_ - so am I missing something?

**Comment by aarppe** [comment]
Concerning non-3s forms being dictionary entries, those are now fortunately coded as IPC's, as follows:

```
cat ~/GDrive/CreeFST/Wolvengrey/Wolvengrey_crk2eng_200202_altlab.tsv | gawk -F"\t" '$0 ~ /[12][sp]/ { print $1, $3, $4; }' | egrep '(kinanâskomitin)|(kitatamihin)|(kôhkomipaninaw)|(tânitahtwasiyêk)'
kinanâskomitin IPJ thank you, I am grateful to you
kitatamihin IPJ thank you, you make me smile, you please me
kôhkomipaninaw NA-2 "cucumber; literally: ""our deceased grandmother"""
tânitahtwasiyêk IPC there are how many of you (in the family)?
```
This now means that they are not treated as verb stems, though they can be recognized productively as inflected forms of the respective lemmas, receiving a translation for the lemma, beside the possible translation for the inflected form itself, cf.

![image](https://user-images.githubusercontent.com/15003407/75747869-4dbc4a00-5cdb-11ea-9dd8-7a50f8df62e6.png)

However, the above example shows that we might want to move the general wordclass label (Noun, Verb, Particle) to the lemma - and it may well be that the inflected form has a different wordclass than the lemma.
